src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -405,20 +427,31 @@ def __new_installation_summary_json(self, installation_packages_json):
         self.__refresh_installation_reboot_status()
 
         # Compose substatus message
+        maintenance_run_id = self.execution_config.maintenance_run_id if self.execution_config.maintenance_run_id is not None else ''
+        errors = self.__set_errors_json(self.__installation_total_error_count, self.__installation_errors)
+        substatus_message = self.__compose_installation_substatus_msg(activity_id=self.execution_config.activity_id, reboot_status=self.__installation_reboot_status,
+            maintenance_window=self.__maintenance_window_exceeded, not_selected=not_selected_patch_count, excluded=excluded_patch_count,
+            pending=pending_patch_count, installed=installed_patch_count, failed=failed_patch_count,
+            installation_packages=installation_packages_json, start_time=self.execution_config.start_time,
```

Comment:
same comment as above for 'installation_packages'

Reply:
done